app/controllers/catalog_controller.rb

```diff
@@ -67,6 +67,26 @@ class CatalogController < ApplicationController
     config.add_facet_field 'dc_rights',   field: 'cho_dc_rights_ssim', limit: true
     config.add_facet_field 'provided_by', field: 'agg_data_provider_ssim', limit: true
 
+    # "administrative"-like facets
+    config.add_facet_field 'data provider', field: 'agg_provider_ssim', limit: true
```

Comment:
We should find another label here, because this could be confusing (compare `agg_data_provider`, which drives the "Provided by" facet).

Reply:
can you suggest one? these are also changeable in the spotlight admin interface, so we don't need to perfect it now either.